injector/src/test/java/edu/ucr/cs/riple/injector/ClassSearchTest.java

```diff
@@ -477,7 +477,17 @@ public void class_search_benchmark_22() {
   }
 
   @Test
-  public void proper_report_for_class_not_found_test() {
+  public void proper_report_in_err_std_for_class_not_found_test() {
+    String expectedErrorMessage =
```

Comment:
Thinking about these long errors, we probably should just be testing a substring. Something like:

```
String expectedErrorMessage = "Could not find class of type: Top-Level with name: NotIncluded"
[...]
assertTrue(err.toString().contains(expectedErrorMessage));
```

Reply:
e8040841a6b3d56eb5d1a8798052acabcd760a11